<ui version="4.0" >
 <class>CRectangularFilterParamsGuiComp</class>
 <widget class="QWidget" name="CRectangularFilterParamsGuiComp" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="FilterWidthLabel" >
     <property name="text" >
      <string>Filter Width</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QSlider" name="FilterWidthSlider" >
     <property name="minimum" >
      <number>0</number>
     </property>
     <property name="maximum" >
      <number>50</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="FilterHeightLabel" >
     <property name="text" >
      <string>Filter Height</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QSlider" name="FilterHeightSlider" >
     <property name="minimum" >
      <number>0</number>
     </property>
     <property name="maximum" >
      <number>50</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QLabel" name="FilterWidthValueLabel" >
     <property name="text" >
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QLabel" name="FilterHeightValueLabel" >
     <property name="text" >
      <string>1</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
